lib/say_cases.rb

```diff
@@ -0,0 +1,46 @@
+class SayCase < OpenStruct
+  def test_name
+    'test_%s' % description.gsub(/[-,]/, '_').downcase.tr(' ', '_')
```

Comment:
Please do @tommyschaefer and we can bring it on in.  (Better to fix the window now than to remember to fix it later.)